examples/schedules_construction/AMA_Masters2024.py

```diff
@@ -0,0 +1,251 @@
+            end=BoxLocation(Height.MID)
+        ),[
+            MBTags.CENTRE,
+            f3amb.loop(r(1/8)),
+            centred(f3amb.roll('2x4', padded=True)), 
+            f3amb.loop(r(-3/8)),
+            centred(f3amb.roll(np.pi*2,line_length=2*107,padded=True)), 
+            f3amb.loop(r(-3/8)),
+            centred(f3amb.roll('2x4', padded=True)),
+            f3amb.loop(r(1/8)),
+            MBTags.CENTRE,
+        ], line_length=150),
+
+#####################################
+    f3amb.create(ManInfo(
+            "half square-roll", "hSl-roll", 2, Position.END,
+            start=BoxLocation(Height.TOP, Direction.UPWIND, Orientation.INVERTED),
+            end=BoxLocation(Height.MID)
+        ),[
+            f3amb.loop(r(1/4)),
+            centred(f3amb.roll(np.pi*2,line_length=2*55,padded=True)),
+            f3amb.loop(r(1/4)),
+        ]),
+
+#####################################
+    f3amb.create(ManInfo(
+            "SLC-Rolls", "slc-rolls", 5, position=Position.CENTRE, 
+            start=BoxLocation(Height.BTM, Direction.DOWNWIND, Orientation.UPRIGHT),
+            end=BoxLocation(Height.BTM)
+        ),[
+           MBTags.CENTRE,
+            f3amb.loop(r(1/8)),
+            centred(f3amb.roll(np.pi,line_length=2*40,padded=True)),
+            f3amb.loop(-r(1/4)),
+            centred(f3amb.roll(np.pi,line_length=2*40,padded=True)),
+            f3amb.loop(r(1/4)),
+            centred(f3amb.roll(np.pi,line_length=2*40,padded=True)),
+            f3amb.loop(-r(1/4)),
+            centred(f3amb.roll(np.pi,line_length=2*40,padded=True)),
+            f3amb.loop(r(1/8)),
+            MBTags.CENTRE, 
+        ], line_length=80),
+
+#####################################
+    f3amb.create(ManInfo(
+            "Figure 9-roll", "fig9-roll", 3, position=Position.END, 
+            start=BoxLocation(Height.TOP, Direction.DOWNWIND, Orientation.UPRIGHT),
+            end=BoxLocation(Height.MID)
+        ),[
+            f3amb.loop(r(1/4)),
+            centred(f3amb.roll(np.pi,line_length=2*60,padded=True)),
+            f3amb.loop(r(3/4)), 
+        ], loop_radius = 40),
+
+#####################################
+    f3amb.create(ManInfo(
+            "Roll Combo", "RollC", 4, position=Position.CENTRE, 
+            start=BoxLocation(Height.MID, Direction.UPWIND, Orientation.UPRIGHT),
+            end=BoxLocation(Height.TOP)
+        ),[
+            centred(f3amb.roll(r([0.25, 0.25, 0.25, -0.25, -0.25, -0.25]), padded=False)), 
+        ]) ,
+
+#####################################
+    f3amb.create(ManInfo(
+            "Stall Turn", "stall", 3, position=Position.END, 
+            start=BoxLocation(Height.BTM, Direction.UPWIND, Orientation.UPRIGHT),
+            end=BoxLocation(Height.BTM)
+        ),[
+            f3amb.loop(r(1/4)),
+            f3amb.line(length=60),
+            f3amb.stallturn(),
+            centred(f3amb.roll([np.pi],line_length=150,padded=True)),
```

Comment:
the centred function is there to indicate to the mandef that a particular element in a manoeuvre should be in the centre of the box. For roll elements the padded parameter (its true by default) adds the lines before and after the roll and ensures that it is centred on the line.